mule-esb-plugin/src/main/java/org/mule/tooling/esb/sdk/ui/SdkVersionSelectionDialog.java

```diff
@@ -11,36 +14,62 @@
 
 public class SdkVersionSelectionDialog<T> extends DialogWrapper
 {
-    private JLabel myLabel;
-    private JComboBox myComboBox;
+    private JLabel versionLabel;
+    private JComboBox versionComboBox;
+    private TextFieldWithBrowseButton destination;
+    private JLabel destinationLabel;
+
+    private JPanel centerPanel;
 
     public SdkVersionSelectionDialog(JComponent parent, String title, String name, List<String> scalaVersions)
     {
         super(parent, false);
         setTitle(title);
-        myLabel = new JLabel(name);
-        myComboBox = new ComboBox(new DefaultComboBoxModel<>(scalaVersions.toArray()));
-        myLabel.setLabelFor(myComboBox);
+        //Create and populate the panel.
+        centerPanel = new JPanel(new SpringLayout());
+
+        versionLabel = new JLabel(name, JLabel.TRAILING);
+        versionComboBox = new ComboBox(new DefaultComboBoxModel<>(scalaVersions.toArray()));
+        versionLabel.setLabelFor(versionComboBox);
+
+        centerPanel.add(versionLabel);
+        centerPanel.add(versionComboBox);
+
+        destination = new TextFieldWithBrowseButton();
+        destination.addBrowseFolderListener("Select new Mule distribution destination", null, null, FileChooserDescriptorFactory.createSingleFolderDescriptor());
```

Comment:
Can we make that by default it still points to user-home/mule-distro. I like it as a default